### PR TITLE
Improve checking tests readability

### PR DESCRIPTION
**Pull request by JRobsonJr**
This pull request simplifies the current implementation of attribute checking tests. Instead of making many manual checks, a list of attributes can be written for an assertion to be run for each one. It also updates tests to fail when objects don't have the tested properties.

### CODE REVIEW

File: test/artists.test.ts [hunk 1]
```diff
@@ -59,7 +56,7 @@ describe('Artist requests', () => {
                 '1WgXqy2Dd70QQOU7Ay074N',
             ]);
 
-            for (let i = 0; i < severalArtistsResponse.length; i++) {
+            for (let i = 0; i < severalArtistsResponse.length; i += 1) {
```
Comment (JoseRenan) [inline test/artists.test.ts]: :thinking: was this change necessary? i++ is better readable I guess

Comment (JRobsonJr) [reply test/artists.test.ts]: The linter recommends us not to use ++.
